src/main/java/gdx/liftoff/ui/panels/AddOnsPanel.java

```diff
@@ -178,7 +178,9 @@ private void createButtons(Table table, Collection<String> names, boolean capita
 
         table.defaults().growX().space(SPACE_SMALL);
         for (String name : names) {
-            Label label = new Label(capitalize ? name.toUpperCase(Locale.ROOT) : name, skin);
+            name = capitalize ? name.toUpperCase(Locale.ROOT) : name;
```

Comment:
We can use the localization properties here, instead of toUpperCase. That should mean we don't need to special-case iOS, also. The id that comes in as `name` should be a valid key to the properties. I can make these changes after merging, it's no problem.